src/core/bridge/resourcebridge.cpp

```diff
@@ -193,24 +193,4 @@ void UnloadResourceByCrc(uint64_t crc) {
 void UnloadResourceDirectory(const char* name) {
     Ship::Window::GetInstance()->GetResourceManager()->UnloadDirectory(name);
 }
-
-void WriteTextureDataInt16ByName(const char* name, size_t index, int16_t valueToWrite, bool now) {
```

Comment:
Removed as writing directly textures should explicitly not be supported.

Reply:
Sounds good to me.